inference-engine/ie_bridges/java/tests/OpenVinoTestRunner.java

```diff
@@ -0,0 +1,16 @@
+import org.intel.openvino.InferRequest;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Result;
+import org.junit.runner.notification.Failure;
+
+public class OpenVinoTestRunner {
+    public static void main(String[] args) {
+        Result result = JUnitCore.runClasses(IECoreTests.class, CNNNetworkTests.class, InferRequestTests.class, InputInfoTests.class, TensorDescTests.class);
```

Comment:
Is there a way to avoid manual addition of classes? So user just adds a new file to `java/tests` directory.